docs/5.2/about/brand/index.html

```diff
@@ -613,40 +615,47 @@ <h5>Community</h5>
 <script src="/docs/5.2/assets/js/docs.min.js"></script>
 
 <script>
-
+  
   document.querySelectorAll('.btn-edit').forEach(btn => {
     btn.addEventListener('click', event => {
       const htmlSnippet = event.target.closest('.bd-code-snippet').querySelector('.bd-example').innerHTML
-      StackBlitzSDK.openBootstrapSnippet(htmlSnippet)
+
+      
+      const classes = Array.from(event.target.closest('.bd-code-snippet').querySelector('.bd-example').classList).join(' ')
+
+      const jsSnippet = event.target.closest('.bd-code-snippet').querySelector('.btn-edit').getAttribute('data-sb-js-snippet')
+      StackBlitzSDK.openBootstrapSnippet(htmlSnippet, jsSnippet, classes)
     })
   })
 
-  StackBlitzSDK.openBootstrapSnippet = snippet => {
+  StackBlitzSDK.openBootstrapSnippet = (htmlSnippet, jsSnippet, classes) => {
     const markup = `<!doctype html>
 <html lang="en">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="https:\/\/cdn.jsdelivr.net\/npm\/bootstrap@5.2.0-beta1\/dist\/css\/bootstrap.min.css" rel="stylesheet">
+    <link href="https:\/\/cdn.jsdelivr.net\/npm\/bootstrap@5.2.0\/dist\/css\/bootstrap.min.css" rel="stylesheet">
+    <link href="https://getbootstrap.com/docs/5.2/assets/css/docs.css" rel="stylesheet">
```

Comment:
@julien-deramond why is this here? Doesn't look good practice to load our docs CSS elsewhere and it should not be needed.

Reply:
More info in https://github.com/twbs/bootstrap/pull/36637. Without it, lots of examples doesn't work in StackBlitz. We should try to improve those examples without having to use extra CSS, and so this `docs.css`, and so this line would disappear.

Reply:
Yeah, the docs.css should definitely not be included in StackBlitz. It's dangerous and a bad practice. We should revert it before the v5.2.0 release.

Reply:
Bad practice, temporarily yeah. Dangerous, not sure why. IMO it is better for the end user to have examples that work. But if you're strongly against it I understand, your call.